fastq_filtrator.py

```diff
@@ -0,0 +1,103 @@
+## Functions
+# This is a function that returns "Pass" if gc content of a read is between gc_bounds and "Fail" if it's not
+def gc_filter(seq, gc_bounds):
+    if len(gc_bounds) == 2:
+        min_gc, max_gc = int(gc_bounds[0]), int(gc_bounds[1])
+    elif len(gc_bounds) == 1:
+        min_gc, max_gc = int(gc_bounds[0]), 0 
+    else:
+        min_gc, max_gc = 0, 100
+    gc_count = int((seq.count('G') + seq.count('C')) * 100 / len(seq))
+    if ((gc_count < max_gc) and (gc_count > min_gc)):
+        return "Pass"
+    else:
+        return "Fail"
+ 
+# This is a function that counts an average qual phred33 for a read and returns "Pass"  if qual is more than a quality threshhold or "Fail" if it's not
+# 1. creating Qlist
+a = []
+for number in range(0,41):
+    a += [number]
```

Comment:
1. Плохое имя для переменной
2. У `range` первый аргумент - 0 по-умолчанию
3. Можно чуть проще
```suggestion
quality_scale = list(range(41)):
```